_pages/documentation/gem5-stdlib/7-using-local-data-sources.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: documentation
+title: Using local Data sources in gem5
+parent: gem5-standard-library
+doc: gem5 documentation
+permalink: /documentation/gem5-stdlib/using-local-resources
+author: Harshil Patel
+---
+
+gem5 supports using local data sources in the form of a MongoDB Atlas and JSON datasource. To utilize data sources other than the main gem5 resources database, you will need to override the gem5-resources-config.
```

Comment:
One more thing: Calling these "local" isn't right. We don't mean local. If they hose their own MongoDB instance on the cloud, that's not local. I'd just phrase things in the style of "Add a gem5 Resources MongoDB Datasource".